_articles/ko/how-to-contribute.md

```diff
@@ -225,149 +225,149 @@ README를 스캔하여 깨진 링크 또는 오타를 찾을 수 있습니다. 
 * [Up For Grabs](http://up-for-grabs.net/)
 * [Contributor-ninja](https://contributor.ninja)
 
-### A checklist before you contribute
```

Comment:
잘 기억이 나지 않습니다만, 부제목에 한글로 번역하지 않은 이유는 해당 쳅터로 넘어오는 링크가 한글이면 인코딩이 깨져서 링크 경로를 찾을 수 없기 때문이었습니다.

I did not translate the subtitle(h2~h3) into Hangul, because I could not access the link to that page.